src/containers/Project/PullRequest/Layouts/LayoutDeveloper.js

```diff
@@ -81,26 +78,33 @@ class LayoutDeveloper extends Component {
   }
 
   render() {
+    const { pullRequest } = this.props
     return (
       <div style={{ padding: '0 20px' }}>
         <Tabs defaultActiveKey={0} id="layout-developer-tabs">
           <Tab style={{ margin: '20px 0' }} eventKey={0} title="Summary">
             <PullRequestSummary
               onAddReviewer={this.addReviewer}
               onToggleReviewers={this.toggleReviewers}
-              reviewers={this.state.reviewers}
+              pullRequest={pullRequest}
               toggleReviewers={this.state.toggleReviewers}
             />
           </Tab>
           <Tab style={{ margin: '20px 0' }} eventKey={1} title={tabTitle('Discussion', 4)}>
-            <PullRequestDiscussion />
+            <PullRequestDiscussion
+              onSaveComment={() => {}}
```

Comment:
We shouldn't create arrows function in render for performance reasons (unless this is just stub) :)